Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java

```diff
@@ -1,327 +1,292 @@
-                jsonString.put("description", description);
-
-                positionsList.add(jsonString);
-
+                company = removeLastWhiteSpace(company);
+                title = removeLastWhiteSpace(title);
+                timePeriod = removeLastWhiteSpace(timePeriod);
+                description = removeLastWhiteSpace(description);
+                JSONObject jsonStringChild = new JSONObject();
+                jsonStringChild.put("company", company);
+                jsonStringChild.put("title", title);
+                jsonStringChild.put("time period", jsonStringService.getLongDuration(timePeriod));
+                jsonStringChild.put("description", description);
+                positionsList.add(jsonStringChild);
                 title = "";
                 description = "";
                 timePeriod = "";
                 location = "";
             }
-
-            if (styleIntigerValue1 == 12) {
-
+            if (styleFontSize == 12) {
                 company += element.text() + " ";
             }
-
-            if (styleIntigerValue1 == 11) {
-
+            if (styleFontSize == 11) {
                 title += element.text() + " ";
-
             }
-
-            if (styleIntigerValue1 == 10 && fontColor.matches("color:#b0b0b0")) {
+            if (styleFontSize == 10 && fontColor.matches("#b0b0b0")) {
                 location += location + " ";
             }
-
-            if (styleIntigerValue1 == 10 && fontColor.matches("color:#181818")) {
-
-                if (beforeFontColor.matches("color:#b0b0b0")) {
-
+            if (styleFontSize == 10 && fontColor.matches("#181818")) {
+                if (previousElementFontSize.matches("#b0b0b0")) {
                     timePeriod = description;
-
                     description = "";
                 }
                 description += element.text() + " ";
             }
         }
-
         fileManageService.discardFiles(savedFileLocation);
         fileManageService.discardFiles(savedFileLocation.substring(0, savedFileLocation.length() - 4) + ".html");
 
-        return jsonOutput.toString();
+        jsonData.put("profile", profileDetails);
+        jsonData.put("experiences", experienceList);
+        finalJsonOutput.put("success", successResponse);
+        finalJsonOutput.put("data", jsonData);
+        return finalJsonOutput.toString();
+    }
+    public String extractStyleValues(Element element, int indexOfAttribute){
+        String style = element.attr("style");
+        String[] styleValues = style.split(";");
+        String styleValue = styleValues[indexOfAttribute];
+        String[] keyAndValuePair = styleValue.split(":");
+        return keyAndValuePair[1];
+    }
+    public String removeLastWhiteSpace(String string){
+        if (string != "" && string.charAt(string.length() - 1) == ' ')
+            string = string.substring(0, string.length() - 1);
+        return string;
+    }
+    public String exceptionHandled(){
```

Comment:
This is not the expected error handler. even the success response needs to go through the handler and construct the response from the handler. before returning the response

Reply:
updated,